### PR TITLE
Build docker images on travis

### PR DESCRIPTION
**Pull request by dmathieu**
Don't rely on DockerHub anymore.

### CODE REVIEW

**Comment by beanieboi** [comment]
works for me :)

File: .travis.yml [hunk 1]
```diff
@@ -2,19 +2,29 @@ sudo: required
 dist: trusty
 language: shell
 services:
-  - docker
```
Comment (tt) [inline .travis.yml]: Why are these no longer indented? [It's the default YAML syntax](http://yaml.org/spec/1.2/spec.html#id2759963).

Comment (dmathieu) [reply .travis.yml]: It was regenerated to use Travis' generator

Comment (edmorley) [reply .travis.yml]: Fwiw prettier now supports yaml (https://prettier.io/blog/2018/07/29/1.14.0.html#yaml)... :-)

Comment (beanieboi) [reply .travis.yml]: I run `travis lint` before I commit changes to `.travis.yml` and it does flatten the structure as well

File: docker-build.sh [hunk 2]
```diff
@@ -5,10 +5,10 @@ set -euo pipefail
 cd "$(dirname "${BASH_SOURCE[0]}")"
 . stack-helpers.sh
 
-[ $# -eq 1 ] || abort usage: $(basename "${BASH_SOURCE[0]}") STACK
+[ $# -eq 2 ] || abort usage: $(basename "${BASH_SOURCE[0]}") STACK
```
Comment (edmorley) [inline docker-build.sh]: `BUILD.md` needs updating to reflect two args must now always be passed. Or alternatively, `IMAGE_TAG` below could have a fallback default.

Comment (dmathieu) [reply docker-build.sh]: Thanks. I'll update that doc.

**Pull request merged**